solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -2157,6 +2169,12 @@ public void waitForLoadingCore(String name, long timeoutMs) {
     solrCores.waitForLoadingCoreToFinish(name, timeoutMs);
   }
 
+  // ---------------- Core admin handler operations --------------
+
+  protected <A> A createCoreAdminHandlerOperation(String operationClass, Class<A> clazz) {
+    return loader.newInstance(operationClass, clazz, null, null, null);
```

Comment:
There's a two-arg `loader.newInstance` variant it seems, wondering if that could be used instead here, or if not leave a comment w.r.t. why this variant is used. Though if the two-arg variant is usable perhaps `createCoreAdminHandlerOperation` would not be needed?

Reply:
Nice catch! Thank you for pointing this out! I just copy/pasted the block from `createHandler()` method withot investigating in more details, my bad :(

Reply:
Done